### DISCUSSION TITLE
Addition of hacktoberfest topic to the repo

### DISCUSSION THREAD

**Discussion by kishaningithub** [go-resty/resty#483] [Ideas]
It would be cooler, if you can add hacktoberfest topic to this repo in the about section.

It would enable contributors to get cool swags(T-Shirts and stickers) :-) 

More info on hacktoberfest:  https://hacktoberfest.digitalocean.com/

### Current list of topics to the repo

<img width="312" alt="Screenshot 2021-10-26 at 4 55 05 PM" src="https://user-images.githubusercontent.com/763760/138868476-33f649cf-0831-4f0a-b477-fef92fab50eb.png">

**Comment by jeevatkm** [comment]
@kishaningithub Thanks for bringing up. I have added `hacktoberfest` and `hacktoberfest2021` topics on the repo. 
Plus I have added `hacktoberfest-accepted` label to the PRs of `v2.7.0` release

Let me know if any other actions are required from my end.
